timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java

```diff
@@ -406,7 +406,7 @@ public void nearbyCommitsDoNotAffectResultsPresentInCache() {
         AtomicReference<LockWatchCache> lwCache = new AtomicReference<>(null);
         PreCommitCondition condition = timestamp -> {
             if (timestamp != startTs.get()) {
-                simulateOverlappingWriteTransaction(lwCache, timestamp);
+                simulateOverlappingWriteTransaction(lwCache, startTs.get(), timestamp);
```

Comment:
Changes in this class are a drive-by to reduce flakes (there was a weird race condition here, because I think other transactions were being started in the background).